composer.json

```diff
@@ -142,7 +142,7 @@
         "drupal/field_group": "3.1",
         "drupal/file_mdm": "1.1",
         "drupal/flag": "4.0-alpha3",
-        "drupal/gin": "3.0-alpha20",
+        "drupal/gin": "^3.0",
```

Comment:
Thanks for updating @agami4 & @ribel 
However I think in this case it's not a good idea to have this caret constrained.
Gin is still in alpha, which means that whenever someone updates / installs Open Social it will grab the latest available 3.0 version without us knowing.
Since it's in alpha it can potentially mean there are still potential backwards compatible breaking changes. 

So imagine that happen when someone runs a `composer update goalgorilla/open_social -W` that means their site will break.
Ideally we lock this on the alpha version which you test it, and caret it once its in a RC / stable release.

Reply:
Hey @ronaldtebrake, thank you for the response. I've added the latest alpha version of the gin theme.